src/reverse_map.ts

```diff
@@ -10,13 +11,26 @@ export async function init()
 		.map(x => x.join('/'))
 		.map(x => ({ category: "crypto", provider: "binance", ident: x, type: "price" as IDataType}))
 		.map(x => ({ ...x, hash: hashDataId(x) }))
-		.reduce((acc, cur) => ({ ...acc, [cur.hash]: cur }), {} as { [key: string]: IDataDefinition })
+		.reduce((acc, cur) => ({ ...acc, [cur.hash]: cur }), dataDefinitions)
 	
 	dataDefinitions["363e7fe8b47534460fd06dafd5e18a542fe1aaa78038d7ca5e84694f99a788e5"] = dataDefinitions[hashDataId({
```

Comment:
hardcoded hash?

Reply:
Data migration. Temporary workaround until we fully move to DB hash storage.